src/devtools/client/debugger/src/components/Editor/LineHitCounts.tsx

```diff
@@ -22,8 +24,21 @@ function LineHitCounts({ cm }: Props) {
     () => (hitCounts !== null ? hitCountsToMap(hitCounts) : null),
     [hitCounts]
   );
-
   const dispatch = useAppDispatch();
+  const uniqueHitCounts = Array.from(
+    new Set(hitCounts ? hitCounts.map(hitCount => hitCount.hits) : [])
+  );
+  const maxHitCount = Math.max(...uniqueHitCounts);
+  const hitCountColorMap = useMemo(
+    () =>
+      new Map(
+        uniqueHitCounts.map(hitCount => [
+          hitCount,
+          Math.min(1, Math.max(0, Math.log(hitCount) / Math.log(maxHitCount))) || 0,
```

Comment:
Totally no big deal! But I am a little curious for your thinking about log vs cube root here! In my experience, log maybe over-emphasizes lower amounts _by a bit_.

Reply:
Yeah, I'm not sure, here is an example of both:

Log            |  Cube Root
:-------------------------:|:-------------------------:
<img width="512" alt="image" src="https://user-images.githubusercontent.com/2762082/174394332-1ee0025d-78dd-48af-ae9e-d76715009faf.png"> | <img width="511" alt="image" src="https://user-images.githubusercontent.com/2762082/174394241-561996d7-337f-45e7-868d-cf634d3e1dee.png">

I probably don't have enough space between the color lightness for the cube root example which is making it hard to discern any difference. Seeing it now, it doesn't really make sense as a monochromatic scheme either and seems worth interpolating between different hues.

Is it apparent the color is mapped to each source file and not the whole project? Like when moving between files and a 4 count hit will having varying color, is that disruptive? Sounds like something we could test amongst ourselves and see how it feels.

Reply:
Yeah, looks to be a little better using different hues with cube root:

<img width="510" alt="image" src="https://user-images.githubusercontent.com/2762082/174398129-ab959ede-e843-4e45-aab5-d7ab44c41b0c.png">

Reply:
I think the advantage of cube root won't be obvious until you're looking at a file with hundreds/thousands of line hits. The difference between 1 and 4 isn't big, so maybe it's actually good that we don't style it super differently? That being said, I'm not sure I understand the difference I'm seeing above. I'd still expect the lowest value to be the same color?

> Is it apparent the color is mapped to each source file and not the whole project? Like when moving between files and a 4 count hit will having varying color, is that disruptive? Sounds like something we could test amongst ourselves and see how it feels.

This is a good question. I think we'll have to try and see how it feels.